src/main/scala/com/cognite/sdk/scala/v1/relationships.scala

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1
+
+import java.time.Instant
+import com.cognite.sdk.scala.common._
+
+final case class Relationship(
+    externalId: String,
+    sourceExternalId: String,
+    sourceType: String,
+    targetExternalId: String,
+    targetType: String,
+    confidence: Option[Double] = None,
+    startTime: Option[Instant] = None,
+    endTime: Option[Instant] = None,
+    createdTime: Instant = Instant.ofEpochMilli(0),
+    lastUpdatedTime: Instant = Instant.ofEpochMilli(0),
+    dataSetId: Option[Long] = None,
+    labels: Option[Seq[CogniteExternalId]] = None
```

Comment:
Nitpick: let's use the same order as the API docs for these fields (even if it doesn't really matter).

Reply:
Sure! I will reorder